README.md

```diff
@@ -1,11 +1,11 @@
-# Action for Installing MATLAB on GitHub-Hosted Runner
+# Action for Setting Up MATLAB on GitHub-Hosted Runner
 
-Before you run MATLAB&reg; code and Simulink&reg; models on a [GitHub&reg;-hosted](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners) runner, first use the [Set Up MATLAB](#set-up-matlab) action. The action installs the specified MATLAB release on a Linux&reg; virtual machine. If you do not specify a release, the action installs the latest release of MATLAB. 
+Before you run MATLAB&reg; code and Simulink&reg; models on a [GitHub&reg;-hosted](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners) runner, first use the [Setup MATLAB](#set-up-matlab) action. The action sets up the specified MATLAB release on a Linux&reg; virtual machine. If you do not specify a release, the action sets up the latest release of MATLAB.
```

Comment:
Won't the link break here?

[Setup MATLAB](#set-up-matlab)

Reply:
RIP I merged right before reading this 😂 
I'm going to follow this PR up with another one 😂

Reply:
Okay I just checked, I think the link still works 🤔 
Just checked in main 🤔